EIPS/eip-walletnft.md

```diff
@@ -0,0 +1,64 @@
+---
+eip: <to be assigned>
+title: NFT with wallet
+description: NFT (ERC721) with only one token minted
+author: Victor Munoz (@victormunoz)
+discussions-to: https://ethereum-magicians.org/t/erc721-minting-only-one-token/8602/2
+status: Draft
+type: Standards
+category (*only required for Standards Track): ERC
+created: 2022-03-25
+requires (*optional): 721
+---
+
+## Abstract
+NFT associated uniquely with a single contract address.
```

Comment:
This needs to be longer and more descriptive. See other Final EIPs for example abstracts.